aptos-move/aptos-e2e-comparison-testing/src/main.rs

```diff
@@ -35,6 +36,29 @@ pub enum Cmd {
         /// Dump the write set of txns
         #[clap(long, default_value_t = false)]
         dump_write_set: bool,
+        /// Target account
```

Comment:
This comment is useless :) Either describe what is the usage (I think that would be the best haha) or delete it.

Reply:
done